src/models.rs

```diff
@@ -150,28 +150,28 @@ pub enum LoginOptions {
 }
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct SignInWithEmailAndPasswordPayload {
-    pub(crate) email: String,
-    pub(crate) password: String,
+pub struct SignInWithEmailAndPasswordPayload<'a> {
+    pub(crate) email: &'a str,
+    pub(crate) password: &'a str,
 }
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct SignInWithPhoneAndPasswordPayload {
-    pub(crate) phone: String,
-    pub(crate) password: String,
+pub struct SignInWithPhoneAndPasswordPayload<'a> {
+    pub(crate) phone: &'a str,
+    pub(crate) password: &'a str,
 }
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct SignUpWithEmailAndPasswordPayload {
-    pub(crate) email: String,
-    pub(crate) password: String,
+pub struct SignUpWithEmailAndPasswordPayload<'a> {
+    pub(crate) email: &'a str,
+    pub(crate) password: &'a str,
     pub(crate) options: Option<SignUpWithPasswordOptions>,
 }
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct SignUpWithPhoneAndPasswordPayload {
-    pub(crate) phone: String,
-    pub(crate) password: String,
+pub struct SignUpWithPhoneAndPasswordPayload<'a> {
+    pub(crate) phone: &'a str,
+    pub(crate) password: &'a str,
     pub(crate) options: Option<SignUpWithPasswordOptions>,
 }
```

Comment:
Not part of this PR but consider making these structs private because they are only used internally as payloads for the http calls.

Reply:
Thanks you for helping polish things up!

CI doesn't like anything pre-merge at the moment, but this looks fine so I'll merge and fix if anything bubbles up.